debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/AbstractPointer.java

```diff
@@ -294,7 +296,8 @@ public String toString() {
 		String pointerType = getClass().getName();
 		try {
 			if ((this instanceof J9ObjectPointer)||(this instanceof J9IndexableObjectPointer) ||
-				(this instanceof J9IndexableObjectContiguousPointer) || (this instanceof J9IndexableObjectDiscontiguousPointer)) {
+				(this instanceof J9IndexableObjectContiguousPointer) || (this instanceof J9IndexableObjectDiscontiguousPointer) ||
+				(this instanceof J9IndexableObjectWithDataAddressContiguousPointer) || (this instanceof J9IndexableObjectWithDataAddressDiscontiguousPointer) ) {
```

Comment:
`||` goes at the beginning of a continued line, `) {` on a line by itself, one type per line.

Reply:
This has not been addressed:
```java
			if ((this instanceof J9ObjectPointer)
				|| (this instanceof J9IndexableObjectPointer)
				|| (this instanceof J9IndexableObjectContiguousPointer)
				|| (this instanceof J9IndexableObjectDiscontiguousPointer)
				|| (this instanceof J9IndexableObjectWithDataAddressContiguousPointer)
				|| (this instanceof J9IndexableObjectWithDataAddressDiscontiguousPointer)
			) {
```